modules/samples/oauth2/custom-grant/src/main/java/org/wso2/sample/identity/oauth2/grant/mobile/MobileGrant.java

```diff
@@ -154,4 +156,9 @@ private boolean isValidMobileNumber(String mobileNumber){
         return false;
     }
 
+    @Override
+    public boolean isOfTypeApplicationUser() throws IdentityOAuth2Exception {
+        return true;
```

Comment:
Is there any harm keeping this?  Even if superclass later change this value, it won't effect as we are explicitly setting it here and in one look the details are visible within the class if we keep it. Anyway I can remove if there is a thorough objection to keep it.. :)